sky/serve/serve_utils.py

```diff
@@ -22,9 +23,11 @@
 from sky import backends
 from sky import exceptions
 from sky import global_user_state
+from sky import sky_logging
 from sky import status_lib
 from sky.serve import constants
 from sky.serve import serve_state
+from sky.skylet import constants as skylet_constants
```

Comment:
how about we align the implementation, e.g. if both sky.skylet.constants and sky.serve.constants are imported, we always use `from sky.serve import constants as serve_constants`